
syntax = "proto3";

import "google/protobuf/empty.proto";

package airmobisim;

service AirMobiSim{
    
    rpc Start (google.protobuf.Empty) returns (google.protobuf.Empty);

    rpc ExecuteOneTimeStep (google.protobuf.Empty) returns (ResponseQuery);
    
    rpc Finish (google.protobuf.Empty) returns (google.protobuf.Empty);
}


message Response{

    int32 id = 1;
    float x = 2;
    float y = 3;
    float z = 4;
}


message ResponseQuery{
    repeated Response responses = 1;
}